react/features/base/participants/actions.js

```diff
@@ -9,75 +9,6 @@ import {
 import { getLocalParticipant } from './functions';
 
 /**
- * Action to update a participant's avatar ID.
```

Comment:
I'm -1 on this change. Each participant property might be changed from different places, so encapsulating it in an action makes sense to me.

IIRC @lyubomir added those, so I'll let him have the last say, but I'd like them to stay separate.